src/node/full_node.cpp

```diff
@@ -221,7 +221,7 @@ void FullNode::start() {
         received_blocks_++;
       }
 
-      if (dag_blk_mgr_->pivotAndTipsAvailable(blk)) {
+      if (dag_mgr_->pivotAndTipsAvailable(blk)) {
```

Comment:
Maybe put check inside of the addDagBlock and have it return a bool. But this is fine as well

Reply:
addDagBlock is used also elsewhere so I think I can not do it